site/en/blog/devtools-digest-cds-edition/index.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'DevTools Digest (CDS Edition): A Glimpse into the Future and RAIL Profiling'
+description: >
+  Learn how DevTools is going mobile first with a new, streamlined Device Mode that’s always on. Use the color buttons to quickly add colors to your selectors and find out what’s coming to DevTools soon.
+authors:
+  - pbakaus
+date: 2015-11-23
+updated: 2015-11-23
+tags:
+  - devtools
+  - digest
+  - update
+---
+
+Learn how DevTools is going mobile first with a new, streamlined Device Mode that’s always on. Use the color buttons to quickly add colors to your selectors and find out what’s coming to DevTools soon.
+
+## A glimpse into the future of authoring
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dJR-n8szgBc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+We’re just coming back from the Chrome Dev Summit where I showed you what working with DevTools looks like today and in the future. I usually don’t mention any features that are still experiments or heavy works of progress in this digest but I’m making an exception this time. If you don’t have time to watch the whole talk, here’s the gist:
+
+### Device Mode version 2
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/r-uc-fENa7w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
These YouTube embeds need to use the shortcode https://developer.chrome.com/docs/handbook/components/#youtube